app/templates/main/catalogue.html

```diff
@@ -2,42 +2,195 @@
+            <strong>Select a subject</strong>
+            <div class="subjects">
+              <ul class="subjects">
+                <li><a href="">Africa</a></li>
+                <li><a href="">Aid and development</a></li>
+                <li><a href="">Air force</a></li>
+                <li><a href="">Air transport</a></li>
+                <li><a href="">Ancient Monuments and Historical Buildings</a></li>
+                <li><a href="">Archives and libraries</a></li>
+                <li><a href="">Armed Forces (General Administration)</a></li>
+                <li><a href="">Army</a></li>
+                <li><a href="">Art, architecture and design</a></li>
+                <li><a href="">Asia</a></li>
+                <li><a href="">Australia and Pacific</a></li>
+              </ul>
+        
+            </div>
+        
+          </div>
+        
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+</div>
 
-{% block content %}
 <div class="tna-section">
   <div class="tna-container">
-    <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
-      <form action="{{ url('search:catalogue') }}" method="get">
-        <h1 class="tna-heading-xl">{{ pageTitle }}</h1>
-        {{ tnaSearchField({
-          'label': 'Search the catalogue',
-          'headingLevel': 2,
-          'headingSize': 'm',
-          'id': 'search',
-          'name': 'q',
-          'value': request.GET.get('q') or ''
-        }) }}
-      </form>
+    <div class="tna-column tna-column--full">
+
+      <div class="development-notice">
+        
+          <h2 class="tna-heading-m">
+            This site is still under development
+          </h2>
+          <p>Read our <a href="">blog post</a> for more information and help us out by providing <a href="">feedback</a>.</p>
+        
+      </div>
+
+    </div>
+    
+  </div>
+
+</div>
+
+<div class="tna-!--margin-top-xs">
+  <div class="tna-container">
+    <h3 class="tna-heading-l">Other ways to explore our collection</h3>
+  </div>
+  <div>
+    
+    <ul class="tna-container tna-container--nested">
+      {% for page in top_pages %}
+        <li class="tna-column tna-column--width-1-3 tna-column--full-tiny tna-!--margin-top-l">
+          {{ tnaCard({
+            'title': page.title,
+            'headingLevel': 3,
+            'headingSize': 's',
+            'href': page.full_url,
+            'imageAlt': '',
+            'imageSrc': page.teaser_image.jpeg.full_url,
+            'imageWidth': page.teaser_image.jpeg.width,
+            'imageHeight': page.teaser_image.jpeg.height,
+            'meta': [
+              {
+                'title': 'Type',
+                'text': 'Explore the collection'
+              }
+            ],
+            'body': page.teaser_text | safe,
+            'fullAreaClick': True
+          }) }}
+        </li>
+      {% endfor %}
+    </ul>
+    <div class="tna-container">
```

Comment:
I think we need to add some space above this with a class of `tna-!--margin-top-l`